code/modules/atmospherics/machinery/components/unary_devices/thermomachine.dm

```diff
@@ -188,13 +232,15 @@
 	min_temperature = T20C
 	circuit = /obj/item/circuitboard/machine/thermomachine/heater
 
-/obj/machinery/atmospherics/components/unary/thermomachine/heater/on
-	on = TRUE
-	icon_state = "heater_1"
-
 /obj/machinery/atmospherics/components/unary/thermomachine/heater/RefreshParts()
 	..()
 	var/L
 	for(var/obj/item/stock_parts/micro_laser/M in component_parts)
 		L += M.rating
 	max_temperature = T20C + (initial(max_temperature) * L) //573.15K with T1 stock parts
+
+/obj/machinery/atmospherics/components/unary/thermomachine/freezer/on
+	on = TRUE
```

Comment:
Please don't remove the icons, they were changed so that you can see that they're on in DM. Make a new `icon_state_off` var or something instead please. :(

Reply:
Fixed in the latest commit.